langages/python/apprendre-python-quoi-lire-quoi-regarder.md

```diff
@@ -0,0 +1,69 @@
+# Apprendre Python, quoi lire, quoi regarder
+
+La documentation officielle, votre principale source de connaissance (aussi dispo en français) : <https://docs.python.org/3/>
+
+Parmi la documentation, voici quelques pages que nous jugeons essentielles à lire :
+
+* <https://docs.python.org/3/library/functions.html>
+* <https://docs.python.org/3/library/stdtypes.html>
+* <https://docs.python.org/3/library/string.html>
+* <https://docs.python.org/3/library/exceptions.html>
+* <https://docs.python.org/3/reference/datamodel.html>
+
+## Cours vidéos
+
+* En profondeur (conseillé) : <https://www.youtube.com/watch?v=HWxBtxPBCAc&list=PLrSOXFDHBtfHg8fWBd7sKPxEmahwyVBkC>
+* Expéditif (on a quelques [critiques](https://docs.drlazor.be/python_graven.md)) : <https://www.youtube.com/watch?v=psaDHhZ0cPs&list=PLMS9Cy4Enq5JmIZtKE5OHJCI3jZfpASbR>
```

Comment:
Le lien de la critique sera a mettre a jour vers l'article sur not a hub je pense